src/ui/preferences-window/tabs/GeneralTab.swift

```diff
@@ -54,18 +55,20 @@ class GeneralTab: NSViewController, PreferencePane {
 
         let grid = GridView([
             startAtLogin,
+            hideMenubarIcon,
             [holdAndPress, shortcuts],
             [checkboxesExplanations, checkboxes],
             [toShowExplanations, toShow],
         ])
         grid.column(at: 0).xPlacement = .trailing
-        [1, 2, 3].forEach { grid.row(at: $0).topPadding = GridView.interPadding }
+        [1, 2, 3, 4].forEach { grid.row(at: $0).topPadding = GridView.interPadding }
```

Comment:
I think here it looks better with `[2, 3, 4]`, as we want the first 2 checkboxes to be together